code/socialDistribution/views.py

```diff
@@ -308,34 +308,40 @@ def author(request, author_id):
 def create(request):
     return render(request, 'create/index.html')
 
+def post(request, author_id, post_id):
+    """
+        Allows user to view a specific post.
+        Mostly useful to share public (unlisted) posts through url
+    """
+    author = get_object_or_404('Author', pk=author_id)
+    post = get_object_or_404('Post', pk=post_id)
+    pass
+
```

Comment:
What does this do? It throws an error since we don't return an HTTP object.

Reply:
Oh sorry, I was gonna address showing unlisted posts with this view function, but I'm gonna cut this pr here.
Michelle will be dealing with displaying the unlisted posts.
I'll remove this function anyways.